prow/jobs/kyma/components/kyma-components-static-checks.yaml

```diff
@@ -425,28 +400,4 @@ presubmits: # runs on PRs
               requests:
                 memory: 200Mi
                 cpu: 80m
-    - name: pull-telemetry-operator-unit-test
```

Comment:
This one was just set as required today and has been fixed, see [here](https://github.com/kyma-project/test-infra/pull/6450). Please keep it. :)